### ISSUE TITLE
[SFSF]Carousel: A11y - ARIA role should be appropriate for the element

### ISSUE THREAD

**Issue by VidhiNPatel** [SAP/ui5-webcomponents#11429]
### Describe the bug

ARIA role should be appropriate for the element

### Reproduction steps

1. Download the axe DevTools - Web Accessibility Testing Chrome extension
2. Go to https://sap.github.io/ui5-webcomponents-react/v2/?path=/docs/layouts-floorplans-carousel--docs 
3. Open the web inspect -> Select Axe DevTool -> Full Page Scan
4. You will see ARIA role should be appropriate for the element issue reported by Axe tool

### Expected Behaviour

https://dequeuniversity.com/rules/axe/4.10/aria-allowed-role?application=AxeChrome

### Screenshots or Videos

<img width="1695" alt="Image" src="https://github.com/user-attachments/assets/ab10207a-5428-42b6-ba47-7ccc48e614fc" />

### UI5 Web Components for React Version

2.9.2

### UI5 Web Components Version

2.9.0

### Browser

Chrome

### Relevant log output

```shell

```

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by nnaydenow** [comment]
Hello @SAP/ui5-webcomponents-topic-rd,

Could you please take a look? Axe tool is reporting the following issue in:

https://sap.github.io/ui5-webcomponents/nightly/play/#eyJpbmRleC5odG1sIjp7Im5hbWUiOiJpbmRleC5odG1sIiwiY29udGVudCI6IjwhLS0gcGxheWdyb3VuZC1mb2xkIC0tPlxuPCFET0NUWVBFIGh0bWw-XG48aHRtbCBsYW5nPVwiZW5cIj5cblxuXG48aGVhZD5cbiAgICBcbiAgICA8c3R5bGU-XG4gICAgICAqOm5vdCg6ZGVmaW5lZCkge1xuICAgICAgICBkaXNwbGF5OiBub25lO1xuICAgICAgfVxuXG4gICAgaHRtbCB7XG4gICAgICBmb3JjZWQtY29sb3ItYWRqdXN0OiBub25lO1xuICAgIH1cbiAgICA8L3N0eWxlPlxuXG4gICAgPG1ldGEgY2hhcnNldD1cIlVURi04XCI-XG4gICAgPG1ldGEgbmFtZT1cInZpZXdwb3J0XCIgY29udGVudD1cIndpZHRoPWRldmljZS13aWR0aCwgaW5pdGlhbC1zY2FsZT0xLjBcIj5cbiAgICA8dGl0bGU-U2FtcGxlPC90aXRsZT5cbiAgICA8bGluayByZWw9XCJzdHlsZXNoZWV0XCIgaHJlZj1cIi4vbWFpbi5jc3NcIj5cbjwvaGVhZD5cblxuPGJvZHkgc3R5bGU9XCJiYWNrZ3JvdW5kLWNvbG9yOiB2YXIoLS1zYXBCYWNrZ3JvdW5kQ29sb3IpXCI-XG4gICAgPCEtLSBwbGF5Z3JvdW5kLWZvbGQtZW5kIC0tPlxuXG4gICAgPHVpNS1jYXJvdXNlbD5cbiAgICAgICAgPGltZyBzcmM9XCJodHRwczovL3NhcC5naXRodWIuaW8vdWk1LXdlYmNvbXBvbmVudHMvbmlnaHRseS9pbWFnZXMvc2FtcGxlMS5qcGdcIiBhbHQ9XCJMYW5kc2NhcGUgMVwiPlxuICAgICAgICA8aW1nIHNyYz1cImh0dHBzOi8vc2FwLmdpdGh1Yi5pby91aTUtd2ViY29tcG9uZW50cy9uaWdodGx5L2ltYWdlcy9zYW1wbGUyLmpwZ1wiIGFsdD1cIkxhbmRzY2FwZSAyXCI-XG4gICAgICAgIDxpbWcgc3JjPVwiaHR0cHM6Ly9zYXAuZ2l0aHViLmlvL3VpNS13ZWJjb21wb25lbnRzL25pZ2h0bHkvaW1hZ2VzL3NhbXBsZTMuanBnXCIgYWx0PVwiQnVsYlwiPlxuICAgIDwvdWk1LWNhcm91c2VsPlxuICAgIDwhLS0gcGxheWdyb3VuZC1mb2xkIC0tPlxuICAgIDxzY3JpcHQgdHlwZT1cIm1vZHVsZVwiIHNyYz1cIm1haW4uanNcIj48L3NjcmlwdD5cbjwvYm9keT5cblxuPC9odG1sPlxuPCEtLSBwbGF5Z3JvdW5kLWZvbGQtZW5kIC0tPlxuIn0sIm1haW4uanMiOnsibmFtZSI6Im1haW4uanMiLCJjb250ZW50IjoiLyogcGxheWdyb3VuZC1oaWRlICovXG5pbXBvcnQgXCIuL3BsYXlncm91bmQtc3VwcG9ydC5qc1wiO1xuLyogcGxheWdyb3VuZC1oaWRlLWVuZCAqL1xuaW1wb3J0IFwiQHVpNS93ZWJjb21wb25lbnRzL2Rpc3QvQ2Fyb3VzZWwuanNcIjsifSwibWFpbi5jc3MiOnsibmFtZSI6Im1haW4uY3NzIiwiY29udGVudCI6ImltZyB7XG4gICAgbWF4LWhlaWdodDogMzByZW07XG59In19

REgards,
Nayden

**Comment by VidhiNPatel** [comment]
Hi @nnaydenow ,

Could you please provide an update on this issue? It's been open for quite some time. Your insights would be greatly appreciated. Thank you!

**Comment by LidiyaGeorgieva** [comment]
Hello @VidhiNPatel, we are working on a fix for this issue and are currently at the stage of finding the best solution with our designers. Our related item is (TEAM JIRA)-3483.
The fix will be for both https://github.com/SAP/ui5-webcomponents/issues/11515 and this issue.

**Comment by VidhiNPatel** [comment]
Thanks for the reply @LidiyaGeorgieva. Do you have any timeline for this fix?

**Comment by VidhiNPatel** [comment]
Hi @LidiyaGeorgieva, it has been over two months since the last update. Could you please provide the timeline for this fix?

**Comment by olannyv** [comment]
The team is working on general Carousel IxD (interaction, ACC roles, keyboard handling) update with (internal Jira)RODOPI-3539 backlog item that should be completed until mid-october the latest

**Comment by olannyv** [comment]
The team is working on general Carousel IxD (interaction, ACC roles, keyboard handling) update with (internal Jira)RODOPI-3539 backlog item that should be completed until mid-october the latest
